plugin/storage/es/spanstore/reader.go

```diff
@@ -137,6 +140,13 @@ func NewSpanReader(p SpanReaderParams) *SpanReader {
 	if p.UseReadWriteAliases {
 		maxSpanAge = rolloverMaxSpanAge
 	}
+	jt, err := jtracer.New("spanstore")
```

Comment:
Once again, we do not create tracers in random places, only in mains. You already used jtracer.New twice, in two mains - that's it, you cannot call it again. You have to pass the tracers you creates in mains down to these components.